rkt/fetch_test.go

```diff
@@ -56,6 +56,22 @@ type serverHandler struct {
 	auth string
 }
 
+func getSecFlags(opts []string, defopts string, fm map[string]int) *SecFlags {
+	bf, err := NewBitFlags(opts, defopts, fm)
+	if err != nil {
+		panic(fmt.Sprintf("fetch-test: problem initializing flags: %s", err))
+	}
+
+	return (*SecFlags)(bf)
+}
+
+var (
+	// Disable image & tls security checks
+	insecParams   = strings.Join([]string{insecureOptions[1], insecureOptions[2]}, ",")
```

Comment:
I would directly use the "image" and "tls" instead of `insecureOptions[1]` and `insecureOptions[2]`, it avoids having to look for their meaning and you can remove the comment above.

Or, if you want to be safer, use `insecureOptions[insecureImage]` and `insecureOptions[insecureTls]`.